ddgr

```diff
@@ -1454,6 +1460,14 @@ class DdgCmd(object):
                 elif cmd == 'x':
                     Result.urlexpand = not Result.urlexpand
                     printerr('url expansion toggled.')
+                elif cmd[0] == 'c' and cmd[2:].isdigit():
```

Comment:
Double checking as you are accessing `cmd[2:]`, what happen if you press `c` (I mean just `c` without any arguments)? Do we get the right error msg?